### PR TITLE
added playground rules

### PR DESCRIPTION
**Pull request by tusharojha**
## Description

Added a doc about playground info, working, code behind inbuilt APIs, etc.

### CODE REVIEW

**Comment by tusharojha** [comment]
@Yung-Beef please review.

File: docusaurus.config.js [hunk 1]
```diff
@@ -32,12 +32,12 @@ module.exports = {
       theme: require('prism-react-renderer/themes/dracula'),
     },
     announcementBar: {
-      id: 'support_us',
+      id: 'playground',
       content:
-        'We are looking to revamp our docs, please fill out <a target="_blank" rel="noopener noreferrer" href="https://forms.gle/2FHGBma3JPvukyeo6">this survey</a> with any suggestions.',
+        'We just launched the Subsocial Playground for developers, Try now <a target="_blank" rel="noopener noreferrer" href="https://docs.subsocial.network/playground">here</a>.',
```
Comment (olehmell) [inline docusaurus.config.js:37]:
```suggestion
        'We just launched the Subsocial Playground for developers, Try now <a target="_blank" rel="noopener noreferrer" href="https://play.subsocial.network">here</a>.',
```

**Review by olehmell — CHANGES_REQUESTED** [review]

File: docs/develop/Playground.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+id: playground
+title: Playground Rules
+displayed_sidebar: developSidebar
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+Subsocial Playground is an easy tool that allows you to run code snippets from our SDK without downloading anything.
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:11]:
```suggestion
Subsocial's Playground is an easy tool that allows you to run code snippets from our SDK without downloading anything.
```

File: docs/develop/Playground.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+The Playground provides a perfect environment to play, test and debug some of your code snippets before building a real project. 
+
+:::info
+
+It's running on our Testnet called Soonsocial. You can get some test tokens from our [discord bot](https://discord.gg/JRTMTtPK).
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:17]:
```suggestion
The Playground is running on our Testnet called Soonsocial. You can get some test tokens from our [Discord bot](https://discord.gg/JRTMTtPK).
```

File: docs/develop/Playground.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+displayed_sidebar: developSidebar
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+Subsocial Playground is an easy tool that allows you to run code snippets from our SDK without downloading anything.
+
+The Playground provides a perfect environment to play, test and debug some of your code snippets before building a real project.
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:13]:
```suggestion
This provides a perfect environment to play, test and debug some of your code snippets before building a real project. 
```

File: docs/develop/Playground.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+The Playground provides a perfect environment to play, test and debug some of your code snippets before building a real project. 
+
+:::info
+
+It's running on our Testnet called Soonsocial. You can get some test tokens from our [discord bot](https://discord.gg/JRTMTtPK).
+
+:::
+
+### Tryout the playground: [Subsocial Playground](https://docs.subsocial.network/playground).
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:21]:
```suggestion
### Try out the [Subsocial Playground](https://docs.subsocial.network/playground).
```

File: docs/develop/Playground.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+## In-built APIs
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:23]:
```suggestion
## Built-in APIs
```

File: docs/develop/Playground.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+It comes with a set of rules & in-built function to help you focus on library methods rather than writing everything from scratch. The following list represents these functions and their code: 
+
+### Subsocial API
+The Subsocial Playground comes with Subsocial SDK's `api` instance. This is use to interact with most of the SDK methods, here's how it's being created underhood:
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:27]:
```suggestion
The Subsocial Playground comes with the Subsocial SDK's `api` instance. This is used to interact with most of the SDK methods. Here is how it's being created under the hood:
```

File: docs/develop/Playground.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+
+```ts
+  const configDetails = {
+    substrateNodeUrl: 'http://127.0.0.1:9944',
+    ipfsNodeUrl: 'http://127.0.0.1:8080'
+  }
+```
+:::caution
+
+Make sure to run local Subsocial & IPFS node before using these configs in your project.
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:66]:
```suggestion
Make sure to run local Subsocial & IPFS nodes before using these configs in your project.
```

File: docs/develop/Playground.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+  const api = api = await SubsocialApi.create({
+    ...configDetails,
+    useServer: {
+      httpRequestMethod: 'get'
+    }
+  })
+```
+
+### IPFS API
+The Subsocial Playground comes with CRUST IPFS API used as `ipfs` instance. This is use to store data related spaces, posts and comments on the network. Let's see how the configuration is set:
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:82]:
```suggestion
The Subsocial Playground comes with the CRUST IPFS API's `ipfs` instance. This is use to store data related spaces, posts and comments on the network. Let's see how the configuration is set:
```

File: docs/develop/Playground.md [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+    ipfsNodeUrl: 'https://crustwebsites.net'
+  })
+
+  ipfs.setWriteHeaders({
+    authorization: 'Basic ' + authHeader
+  })
+```
+
+### Keyring
+The Subsocial Playground comes with a basic `keyring` instance to help you sign and send transactions. By default, we use the Alice (Test) account for our Testnet transactions but you can change it in our Playground.
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:100]:
```suggestion
The Subsocial Playground comes with a basic `keyring` instance to help you sign and send transactions. By default, we use the Alice (Test) account for our Testnet transactions, but you can change it if you want. 
```

File: docs/develop/Playground.md [hunk 11]
```diff
@@ -0,0 +1,192 @@
...
+```typescript
+  import { Keyring } from "@polkadot/api"
+  import { waitReady } from '@polkadot/wasm-crypto';
+
+  await waitReady()
+  const keyring = new Keyring({ type: 'sr25519' })
+```
+
+### Sending Transaction
+The Subsocial Playground allows you to sign and send the transaction through Mnemonic using Keyring and Polkadot.js extension both.
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:112]:
```suggestion
The Subsocial Playground allows you to sign and send transactions with Mnemonic phrases using both Keyring and the Polkadot.js extension. 
```

File: docs/develop/Playground.md [hunk 12]
```diff
@@ -0,0 +1,192 @@
...
+
+  await waitReady()
+  const keyring = new Keyring({ type: 'sr25519' })
+```
+
+### Sending Transaction
+The Subsocial Playground allows you to sign and send the transaction through Mnemonic using Keyring and Polkadot.js extension both. 
+
+#### Using Keyring
+To sign and send transactions using Keyring through Mnemonic, use:
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:115]:
```suggestion
To sign and send transactions with Mnemonic phrases through Keyring, use:
```

File: docs/develop/Playground.md [hunk 13]
```diff
@@ -0,0 +1,192 @@
...
+      showToast(JSON.stringify(result));
+    } else {
+      console.log('⏱ Current tx status:', status.type);
+      showToast(`⏱ Current tx status: ${status.type}`);
+    }
+  }
+```
+
+#### Using Polkadot.js Extension
+Signing using the polkadot.js extension:
```
Comment (Yung-Beef) [inline docs/develop/Playground.md:152]:
```suggestion
Signing using the Polkadot.js extension: 
```

**Pull request merged**